src/commands/demo-config/config-template.json

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
let's rename this file to what it is really, it's a readetrust config file for covering letter and bill of lading

Reply:
re structured this file to be a generic config file